### PR TITLE
Document ECS supported compose attributes

### PR DESCRIPTION
**Pull request by ndeloof**
**What I did**
Document ECS support for compose attributes as suggested by https://github.com/docker/compose-cli/issues/839

**(not mandatory) A picture of a cute animal, if possible in relation with what you did**

### CODE REVIEW

File: docs/ecs-compose-features.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+| service.tmpfs                  | x |  Not supported on Fargate, see https://github.com/docker/compose-cli/issues/839
+| service.entrypoint             | ✓ | 
+| service.env_file               | ✓ |
+| service.environment            | ✓ |
+| service.expose                 | x |
+| service.extends                | ✓ |
+| service.external_links         | x |
+| service.extra_hosts            | x |
+| service.group_add              | x |
+| service.healthcheck            | ✓ |
```
Comment (gtardif) [inline docs/ecs-compose-features.md:49]: I thought we could not support healthchecks on ECS? (cf https://docker.slack.com/archives/C0111G2RCE4/p1600870550019100). Did I misundestand this ?

File: docs/ecs-compose-features.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+secrets:
+  mysecret:
+    file: ./my_secret1.txt
+```
+
+
+## Container Resources
+
+CPU and memory reservations and limits can be set in compose.
```
Comment (gtardif) [inline docs/ecs-compose-features.md:165]: Even if it's standard compose file format, having an example here would be useful (or a link users can click to see compose file reference on CPU/memory limits and some example)

**Review by gtardif — APPROVED** [review]
1 minor point, 1 question, otherwise LGTM

File: docs/ecs-compose-features.md (hunk 1 above)
Comment (ndeloof) [reply docs/ecs-compose-features.md:49]: we support container healthcheck (same a docker) but not Load Balancer healthcheck
and please don't ask me why ALB don't just rely on container healthchecks that are used to report service health on ECS console :P

**Pull request merged**